src/OmniSharp.Roslyn/OmniSharpWorkspace.cs

```diff
@@ -202,6 +236,31 @@ public override bool CanApplyChange(ApplyChangesKind feature)
             return true;
         }
 
+        internal bool FileBelongsToProject(string fileName, Project project)
+        {
+            if (string.IsNullOrWhiteSpace(project.FilePath) ||
+                string.IsNullOrWhiteSpace(fileName))
+            {
+                return false;
+            }
+
+            var fileDirectory = new FileInfo(fileName).Directory;
+            var projectPath = project.FilePath;
+            var projectDirectory = new FileInfo(projectPath).Directory.FullName;
+
+            while (fileDirectory != null)
+            {
+                if (string.Equals(fileDirectory.FullName, projectDirectory, StringComparison.Ordinal))
```

Comment:
`StringComparison.OrdinalIgnoreCase` is probably a better choice.

Reply:
Sure can do. I was just trying to replicate the logic done in the [BufferManager](https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.Roslyn/BufferManager.cs#L195)